### ISSUE TITLE
Where is the database ?

### ISSUE THREAD

**Issue by lucasvv117** [raminmr/free-api-worldcup2022#67]
Hello, I am kind of new to using APIs, but where is the data source ?  
I mean I get how we can get it using the API, but from where do we get the data ? 
Other question, what request do we need to get the list of all the match of the word cup ?

**Comment by tuminzee** [comment]
@lucasvv117 you can call the api and get the data
Data source is not exposed to the users
To get the list of matches you can call 
Http Metod : GET http://api.cup2022.ir/api/v1/match
A request on Matches endpoint returns all Matches information


To call the API from your application/client refer to
https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API
https://axios-http.com/docs/intro

Before making these calls do read the https://github.com/raminmr/free-api-worldcup2022#readme
It solves most of the doubts

**Comment by abedjaber** [comment]
If I understood you correctly, reading the data requires some technical skills (for example: .Net) by creating an httpclient and proceeding with the rest of the steps and deserialization.

**Comment by tuminzee** [comment]
@abedjaber 
Yes that is correct, although  you can also interact with it by using an API Platform like [**Postman**](https://www.postman.com/)
export of postman collection can be found here https://github.com/raminmr/free-api-worldcup2022/tree/main/collection
